_posts/2019-12-10-variational-gaussian-process/vgp.Rmd

```diff
@@ -0,0 +1,549 @@
+---
+title: "Gaussian Process Regression with tfprobability"
+description: > 
+ Continuing our tour of applications of TensorFlow Probability (TFP), after Bayesian Neural Networks, Hamiltonian Monte Carlo and State Space Models here we show an example of Gaussian Process Regression. In fact, what we see is a rather "normal" Keras network, defined and trained pretty much the usual way, with TFP's Variational Gaussian Process layer pulling off all the magic. 
+ 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019vgp
+bibliography: bibliography.bib
+date: 12-10-2019
+categories:
+  - Probability and statistics
+  - Keras
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/kernel_cookbook.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+
+How do you motivate, or come up with a story around, Gaussian Process Regression on a blog primarily dedicated to deep learning? 
+
+Easy. As demonstrated by seemingly-unavoidable, reliably-recurring twitter "wars" surrounding AI, nothing attracts attention like controversy and antagonism. So, let's go back twenty years and find citations of people saying, "here come Gaussian Processes, we don't need to bother with those finicky, hard to tune neural networks anymore!". And today, here we are; everyone knows _something_ about deep learning but who's heard of Gaussian Processes?
```

Comment:
You know I'm a huge fan of hyphenated modifiers, but you actually don't need them for "seemingly unavoidable" or "reliably recurring," since "seemingly" and "reliably" are adverbs!

Reply:
ok! although in the case of `reliably-recurring`, it seemed to stress the alliteration ;-)))